processing-service-container/pom.xml

```diff
@@ -120,6 +120,7 @@
         <dependency>
             <groupId>com.github.cafdataprocessing</groupId>
             <artifactId>corepolicy-database</artifactId>
+            <version>1.2.0-SNAPSHOT</version>
```

Comment:
Update the `caf.corepolicy.version` token in the main pom instead of adding this.  We don't want to end up dependent on two different versions of core policy.